Firestore/Example/Tests/Local/FSTLRUGarbageCollectorTests.mm

```diff
@@ -155,7 +157,7 @@ - (int)removeOrphanedDocumentsThroughSequenceNumber:(ListenSequenceNumber)sequen
 - (FSTQueryData *)nextTestQuery {
   TargetId targetID = ++_previousTargetID;
   ListenSequenceNumber listenSequenceNumber = _persistence.currentSequenceNumber;
-  FSTQuery *query = FSTTestQuery(absl::StrCat("path", targetID));
+  core::Query query = Query(absl::StrCat("path", targetID));
```

Comment:
You could std::move query on the next line.

Reply:
Done.